app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
```

Comment:
What if we want to create Animal instance and we want it to be `hidden`?

Define hidden in arguments of `__init__` method and give a default value to it